File: findMaxConsecutiveOnes.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int maxOnes = 0, ones = 0;
        for(int i = 0; i < nums.length; i++)
        {
            if(nums[i] == 1)
            {
                ones++;
            }
            else
            {
                ones = 0;
            }
            maxOnes = Math.max(maxOnes, ones);
        }
        
        return maxOnes;
        
    }
}

File: Arrays101/findMaxConsecutiveOnes.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        maxOnes = 0
        ones = 0
        for i in nums:
            if(i == 1):
                ones+=1
            else:
                ones = 0
            maxOnes = max(maxOnes, ones)
            
        return maxOnes
        
